src/auth/jwtOptions.ts

```diff
@@ -0,0 +1,34 @@
+import { expressJwtSecret, SigningKeyNotFoundError } from 'jwks-rsa';
+import type { Algorithm } from 'jsonwebtoken';
+import type { GetVerificationKey } from 'express-jwt';
+
+const authServerRealm = process.env.AUTH_SERVER_REALM_URL
+  ?? 'https://set-AUTH_SERVER_REALM_URL-env-var/auth/realms/pdc';
+const cache = process.env.JWKS_CACHE_AUTH_SERVER_CERTS === 'true';
+const rateLimit = process.env.JWKS_USE_RATE_LIMIT === 'true';
+const jwksRequestsPerMinute = Number(process.env.JWKS_REQUESTS_PER_MINUTE ?? 5);
+const jwksUri = process.env.AUTH_SERVER_JWKS_URI ?? `${authServerRealm}/protocol/openid-connect/certs`;
+
+export const jwtOptions = {
+  // Because of different types from version-to-version in express-jwt (see
+  // jwks-rsa notes), cast to the later express-jwt API's GetVerificationKey.
+  secret: expressJwtSecret({
+    cache,
+    rateLimit,
+    jwksRequestsPerMinute,
+    jwksUri,
+    handleSigningKeyError: (err, cb) => {
+      if (err instanceof SigningKeyNotFoundError) {
+        cb(new Error('Could not fetch certificates from JWKS URI'));
+        return;
+      }
+
+      cb(err);
+    },
```

Comment:
Can you say more about this? Why is this better than the default?

Reply:
I am trying to find where I got this (or why I put it there), but looking again, I do not see why it should be included. The only possible advantage we could get would be if we used our own Error type but we do not. I will see if anything bad happens when removing it.